app/main.py

```diff
@@ -1,3 +1,16 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable
+
+
+def cache(func: Callable) -> Callable:
+    caching_place = {}
+
```

Comment:
You could consider using `wraps` decorator from `functools` here, read more about its advantages in the documentation.